monitoring/tests/system/test_vpcsc_v3.py

```diff
@@ -233,13 +201,13 @@ def test_update_group(self):
         PROJECT_OUTSIDE is None,
         reason="Missing environment variable: GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT",
     )
-    def test_create_metric_descriptor(self):
-        client = monitoring_v3.MetricServiceClient()
-        name_inside = client.project_path(PROJECT_INSIDE)
-        delayed_inside = lambda: client.create_metric_descriptor(name_inside, {})
-        name_outside = client.project_path(PROJECT_OUTSIDE)
-        delayed_outside = lambda: client.create_metric_descriptor(name_outside, {})
-        TestVPCServiceControlV3._do_test(delayed_inside, delayed_outside)
```

Comment:
~~Why does this PR delete this test?~~ I see that it is moved below:  is there a reason for reordering the testcases here?